src/coreclr/CMakePresets.json

```diff
@@ -164,6 +176,60 @@
         "Checked",
         "ARM64"
       ]
+    },
+    {
+      "name": "osx.x64.Debug",
+      "displayName": "osx.x64.Debug",
+      "inherits": [
+        "osx-base",
+        "Debug",
+        "x64"
+      ]
+    },
+    {
+      "name": "osx.x64.Release",
+      "displayName": "osx.x64.Release",
+      "inherits": [
+        "osx-base",
+        "Release",
+        "x64"
+      ]
+    },
+    {
+      "name": "osx.x64.Checked",
+      "displayName": "osx.x64.Checked",
+      "inherits": [
+        "osx-base",
+        "Checked",
+        "x64"
+      ]
+    },
+    {
+      "name": "osx.arm64.Debug",
+      "displayName": "osx.arm64.Debug",
+      "inherits": [
+        "osx-base",
+        "Debug",
+        "ARM64"
+      ]
+    },
+    {
+      "name": "osx.arm64.Release",
+      "displayName": "macos.arm64.Release",
```

Comment:
Display name should be consistent. Since `osx` is still used widely in the codebase and the output directory, and it should probably used for display name.

Reply:
Sure, nice spot.